ms2deepscore/BinnedSpectrum.py

```diff
@@ -0,0 +1,39 @@
+class BinnedSpectrum:
```

Comment:
modules in python are `snake_case` not `CamelCase`

Reply:
That's right, but classes are CamelCase. Or did you mean another part?